imagespace/README.md

```diff
@@ -41,8 +42,11 @@ tangelo --port 9220
 ### Girder with imagespace plugin
 
 1. First ensure you have the proper [Girder_prerequisites](http://girder.readthedocs.org/en/latest/prerequisites.html),
+ - To build on MacOS you will need Xcode if you don't have it already. https://developer.apple.com/xcode/
+ - Check if `xcode-select -p` is pointing to Xcode app Developer directory else point xcode-select to the Xcode app Developer directory using the following command: `sudo xcode-select -s /Applications/Xcode.app/Contents/Developer`
+ - Use latest version of node 6.x 
```

Comment:
This last part seems like an artifact specific to your machine. I would get rid of this entire paragraph since the Girder docs show how to install Girder on MacOS. (If Girder doesn't install on MacOS then changes to docs should be made upstream, though I'm told the current MacOS instructions for Girder do work).